testing/environments/ports.yml

```diff
@@ -11,7 +11,3 @@ logstash:
 kibana:
   ports:
     - "5601:5601"
-
-beat:
```

Comment:
@monicasarbu I should have added a comment here why it is not in the base.yml as I had it there first and it is not obvious. Could you add a comment?